URA-695/reproducibility.py

```
@@ -0,0 +1,181 @@
+    job_run = dxpy.DXApp(name="sentieon-tnbam", alias=alias).run(
+        job_input,
+        project=project_id,
+        folder=folder_path,
+        name=job_name)
+    
+    # return the job_id
+    return job_run.describe(fields={"id":True})["id"]
+    
+    
+
+def main():
```

Ruff (0.8.0)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)